projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -13,6 +14,9 @@ function activeZoneFilter(this: TuiDropdownContextDirective, target: Element): b
     return !this.activeZone.contains(target);
 }
 
+const tuiLongTapDelay = 700;
```

Comment:
nitpicking: Prefix is not necessary since it is not exported. We also write such constants in caps.